server.js

```diff
@@ -45,13 +58,14 @@ app.use((request, response, next) => {
 app.get('/', async (request, response) => {
   const readFile = util.promisify(fs.readFile);
   const md = (await readFile('./README.md', {encoding: 'utf-8'}))
-      .replace(/puppeteeraas.com/g, request.get('host'));
+      .replace(/puppeteeraas\.com/g, request.get('host'));
```

Comment:
Drive-by fix: `.` matches any character except newlines, but in this case we want to match a literal `.`.